### PR TITLE
Rename riff-version to invoker-version

### LINKED ISSUE #414: Change 'riff-version' flag to 'invoker-version'

### PR DESCRIPTION
**Pull request by trisberg**
Fixes #414

### CODE REVIEW

File: riff-cli/cmd/version.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func Version() *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 
 			fmt.Printf("riff CLI version: %v\n", global.CLI_VERSION)
-			fmt.Printf("riff function invoker version: %v\n", global.RIFF_VERSION)
+			fmt.Printf("riff function invoker version: %v\n", global.INVOKER_VERSION)
```
[the remark this answers is not in the document]
Comment (trisberg) [reply riff-cli/cmd/version.go]: You mean "riff function invoker version default"?

[the remark this answers is not in the document]
Comment (trisberg) [reply riff-cli/cmd/version.go]: or "default riff function invoker version"?

[the remark this answers is not in the document]
Comment (ericbottard) [reply riff-cli/cmd/version.go]: I guess the latter

**Comment by ericbottard** [comment]
Merged as squashed commit 4250378e

**Pull request closed without merging**